daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1410,6 +1410,7 @@ private[lf] object SBuiltin {
         throw SpeedyHungry(
           SResultNeedPackage(
             tyCon.packageId,
+            language.Reference.Exception(tyCon),
```

Comment:
Can we ever hit this? I thought loading the packages during preprocessing should be enough to make sure we never run into this so this is an internal error.

Reply:
In off-ledger mode, there is no preprocessing and the validation can be skipped. 
So we can change that to internal error, assuming off-ledger engines have to preload everything.

Reply:
off-ledger engines should already preload everything I believe so that seems like a reasonable requirement. Doesn’t have to be in this PR though

Reply:
In this case it is probably fine to skip completely the check and wait the lookup of the definition fails. 
Since the ref of the definition is a `ExceptionMessageDefRef` the error message will get from `com.daml.lf.speedy.Speedy.Machine#lookupVal` should be clear enough.